### PR TITLE
KFSPTS-19556: Prevent creator on submit and approver on approve from routing I Want edoc to themselves by stopping their ability of entering their own netid in the Approver text field.

### PR DESCRIPTION
**Pull request by nkimble**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import edu.cornell.kfs.module.purap.document.validation.event.AddIWantItemEvent;
 import edu.cornell.kfs.module.purap.util.PurApFavoriteAccountLineBuilderForIWantDocument;
 import edu.cornell.kfs.sys.util.ConfidentialAttachmentUtil;
+
 import org.apache.commons.lang.StringUtils;
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java:15]: we should switch to use org.apache.commons.lang3.StringUtils

File: src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java [hunk 2]
```diff
@@ -608,7 +609,47 @@ public ActionForward deleteAccount(ActionMapping mapping, ActionForm form, HttpS
...
+        }
+        return false;
+    }
+    
+    /**
+     * Prevent IWNT document approver from entering their own NetID 
+     * as the "Approver's NetID" when performing Approve.
+     * 
+     * @return
+     */
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java:638]: similarly, do we need these comments?

File: src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java [hunk 3]
```diff
@@ -743,14 +790,18 @@ public ActionForward sendAdHocRequests(ActionMapping mapping, ActionForm form, H
     @Override
     public ActionForward approve(ActionMapping mapping, ActionForm form, HttpServletRequest request,
             HttpServletResponse response) throws Exception {
+        
+        //Prevent IWNT document approver from entering their own NetID 
+        //as the "Approver's NetID" when performing Approve.
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java:795]: I think these comments could be removed as well

**Review by jhulslander — CHANGES_REQUESTED** [review]
Changes look good, just a couple of small changes: we can switch this class to use the modern StringUtils library, and I am not sure if we need to keep the comments

**Comment by nkimble** [comment]
This is ready to be reviewed again.

**Review by jhulslander — APPROVED** [review]
Changes look good, I shall merge when the unit tests pass

**Pull request merged**